tests/utils-tests.js

```diff
@@ -111,7 +111,7 @@ describe('Utils', () => {
       expect(Utils.isObject(1)).not.toBe(true)
       expect(Utils.isObject('something')).not.toBe(true)
       expect(Utils.isObject(false)).not.toBe(true)
-      expect(Utils.isObject(void 0)).not.toBe(true)
```

Comment:
`void 0` === `undefined` - as per (http://stackoverflow.com/a/4806335/3474514)
It got caught by the eslint option: `"no-void": 1,`